.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        jupyterlab: ['2.0', '3.0.0']
+        jupyterlab: ['3.0.0', '3.1.0', '3.2.0']
```

Comment:
It was nice to check that it was working for all minors of JLab 3 but it might be more reasonable to only check for the latest version in the future ?

Reply:
It can also test the latest and the oldest supported.